avocado/plugins/spawners/podman.py

```diff
@@ -96,19 +127,15 @@ async def spawn_task(self, runtime_task):
         this_path = os.path.abspath(__file__)
         base_path = os.path.dirname(os.path.dirname(os.path.dirname(this_path)))
         avocado_runner_path = os.path.join(base_path, 'core', 'nrunner.py')
-        try:
-            # pylint: disable=E1133
-            proc = await asyncio.create_subprocess_exec(
-                podman_bin,
-                "cp",
-                avocado_runner_path,
-                "%s:%s" % (container_id, entry_point_cmd))
-        except (FileNotFoundError, PermissionError):
+        if not await self._copy_to_image(avocado_runner_path,
+                                         entry_point_cmd, container_id):
             return False
 
-        await proc.wait()
-        if proc.returncode != 0:
-            return False
+        if task.runnable.kind == "avocado-instrumented":
```

Comment:
IIUC, this block is about changing the location of the test files, so that they can be run from `/tmp`, where they were copied to.  Is that right?